build-system/tasks/runtime-test/helpers.js

```diff
@@ -166,6 +178,34 @@ function getJsFilesFor(cssFile, cssJsFileMap) {
   return jsFiles;
 }
 
+function getUnitTestsToRun(unitTestPaths) {
```

Comment:
Since this function is only used by unit tests, move to `unit.js`?

Reply:
Will clean it up once I delete `runtime-test/index.js` which still uses this